code/modules/power/singularity/emitter.dm

```diff
@@ -102,9 +120,26 @@
 	if(stat & (BROKEN))
 		return
 	if(src.state != 2 || (!powernet && active_power_usage))
-		src.active = 0
+		src.active = FALSE
+		src.safety = FALSE //Don't bother with the targeting laser if we're unpowered.

```

Comment:
wouldn't that mean if the safety is _off_, and then it loses power, the safety becomes _on_ (and then flicks on and off every process() tick)?

Reply:
`toggle_safety()` is... maybe not the best term anymore. Per the truth table below I'm basically positing that it update the state based on several factors rather than explicitly toggle